base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -519,6 +518,27 @@ impl ConsensusConstants {
             emission_decay: &ESMERALDA_DECAY_PARAMS,
             emission_tail: 800 * T,
             max_randomx_seed_height: 3000,
+            proof_of_work: algos.clone(),
+            faucet_value: (10 * 4000) * T,
+            transaction_weight: TransactionWeight::v1(),
+            max_script_byte_size: 2048,
+            input_version_range: input_version_range.clone(),
+            output_version_range: output_version_range.clone(),
+            kernel_version_range: kernel_version_range.clone(),
+            permitted_output_types: Self::current_permitted_output_types(),
+        },ConsensusConstants {
+            effective_from_height: 19000,
+            coinbase_lock_height: 6,
+            blockchain_version: 1,
+            valid_blockchain_version_range: 0..=0,
```

Comment:
Doesn’t this need to be updated to include version 1?

Reply:
good catch